automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -191,6 +191,13 @@ class WorkspaceApiSpec
 
       "to get an error message when they try to create a workspace with a bucket region that is invalid" ignore {
         implicit val token: AuthToken = ownerAuthToken
+        // Note that this invalid region passes the regexp in `withWorkspaceBucketRegionCheck`, so workspace creation is
+        // attempted and fails with the bucket creation error. However, due to bug WOR-296, `withTemporaryBillingProject`
+        // is unable to delete the temporary project, causing the test to fail (when this test was first introduced,
+        // there was a different way of creating test workspaces that did not involve creating and deleting test billing
+        // projects). There is already test coverage of `withWorkspaceBucketRegionCheck` in `WorkspaceApiServiceSpec`,
+        // so there is no point in changing the invalid region name to one detected by the regexp, as that would change
+        // the code path executed (no workspace creation would be attempted) and simply duplicate existing unittests.
         val invalidRegion = "invalid-region1"
```

Comment:
[WOR-296](https://broadworkbench.atlassian.net/browse/WOR-296) is the bug to make sure that orphaned resources are not left behind if workspace creation fails